tsconfig.json

```diff
@@ -1,18 +1,19 @@
 {
   "compilerOptions": {
-    "target": "es5",
+    "target": "es2017",
```

Comment:
This would cause us not able to run browsers like IE11 anymore, is this really what we want?

Reply:
If you want to use newer JavaScript then you need to define a lib instead of changing the output (target)

Reply:
The WebdriverIO code only runs on Node.js. We have some scripts in the webdriverio package that we inject into the browser but they should not be compiled.